packages/components/Callout/src/Callout.types.ts

```diff
@@ -26,6 +26,8 @@ export type DirectionalHint =
   | 'bottomCenter'
   | 'bottomRightEdge';
 
+export type DismissBehaviors = 'preventDismissOnKeyDown' | 'preventDismissOnClickOutside';
```

Comment:
Would defining a third behavior 'preventAllDismissal' cover all possible behaviors?? That way users don't have to pass in an array and just a string value.

Reply:
Nah.  I don't want to suggest that a truly persistent callout is currently available.

Reply:
it'd still need to be an array anyway because these behaviors can be individually relevant

Reply:
and there'll be more later